ApplyToTheIndustry/Assets/Scripts/AI Grader/Grader.cs

```diff
@@ -0,0 +1,105 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Grader : MonoBehaviour
+{
+     Dictionary<string, int> skillPoints = new Dictionary<string, int>();
+     HashSet<string> countedSkills = new HashSet<string>();
+     int Totalpoints =0;
+
+    void Start()
+    {
+        
+        GetListOfSkills();
+
+    }
+
+    void Update()
+    {
+        //Checking the job type
+        if (JobContainer.Instance.currentPosting.name == "Job Posting for Programmer")
+        {
+            PointSystemForJobType1();
+
+        }
+
+        CalculateTotalPoints();
+
+        GetFeedback();
```

Comment:
This makes things very hard to read since it constantly spams console. We should likely opt for giving feedback on an event-based call. Something like calling upon this method once the player has pressed a button to confirm their resume.